pkg/graveler/errors.go

```diff
@@ -34,13 +34,15 @@ var (
 	ErrCommitNotFound               = fmt.Errorf("commit %w", ErrNotFound)
 	ErrCreateBranchNoCommit         = fmt.Errorf("can't create a branch without commit")
 	ErrRepositoryNotFound           = fmt.Errorf("repository %w", ErrNotFound)
+	ErrAddressTokenNotFound         = fmt.Errorf("address token %w", ErrNotFound)
```

Comment:
Can we batch the errors together?

Reply:
what do you mean?

Reply:
Put all the addressToken errors together one after the other

Reply:
done